packages/shared/lib/services/sync/run.service.ts

```diff
@@ -169,7 +175,7 @@ export default class SyncRun {
         this.writeToDb = config.writeToDb;
         if (config.writeToDb) {
             this.slackNotificationService = config.slackService;
-            this.activityLogId = config.activityLogId;
+            this.activityLogId = config.activityLogId as unknown as number;
```

Comment:
This cast mixed with the one below that casts it to a string is confusing

Reply:
true, it's only temporary during the transition but let me change that